test/test_images/probe_helper/probe_helper.go

```diff
@@ -41,16 +51,28 @@ type envConfig struct {
 	Timeout int `envconfig:"TIMEOUT_MINS" default:"30"`
 }
 
-func forwardFromProbe(ctx context.Context, c cloudevents.Client, receivedEvents map[string]chan bool, timeout int) cloudEventsFunc {
+func forwardFromProbe(ctx context.Context, sc cloudevents.Client, psc cloudevents.Client, receivedEvents map[string]chan bool, timeout int) cloudEventsFunc {
 	return func(event cloudevents.Event) protocol.Result {
 		log.Printf("Received probe request: %+v \n", event)
 		eventID := event.ID()
 		receivedEvents[eventID] = make(chan bool, 1)
 		ctx, cancel := context.WithTimeout(ctx, time.Duration(timeout)*time.Minute)
 		defer cancel()
-		if res := c.Send(ctx, event); !cloudevents.IsACK(res) {
-			log.Printf("Error when sending event %v to broker: %+v \n", eventID, res)
-			return res
+		switch event.Type() {
+		case "broker-e2e-delivery-probe", "broker-e2e-delivery-probe-kubectl-exec", "broker-e2e-delivery-probe-kubectl-exec-warm-up":
```

Comment:
We can change the event type to just "broker-e2e-delivery-probe" and "cloudpubsubsource-probe". We could use other attributes such as Ce-Source to further differentiate probes of the same event type if needed

Reply:
Done.